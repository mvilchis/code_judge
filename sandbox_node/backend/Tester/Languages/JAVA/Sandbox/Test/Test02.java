
public class Test02{
	public static void main(String[] args) {
		System.out.print("print\n");
		System.out.println("println");
		System.out.append("append\n");
		System.out.write(64);
		System.out.printf("%d %.1f %s",1, 1.1f, "printf");
	}
}
