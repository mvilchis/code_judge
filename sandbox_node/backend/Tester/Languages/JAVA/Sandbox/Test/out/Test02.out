printprintlnappend@11.1printf